packages/create-redwood-app/template/api/db/seed.js

```diff
@@ -12,11 +12,31 @@ async function main() {
   // Ideally this file should be idempotent: running it multiple times
   // will result in the same database state (usually by checking for the
   // existence of a record before trying to create it). For example:
-  //
```

Comment:
I just realized that 

createMany is not supported by SQLite

therefore we will still need the singular:

`await db.user.create`

examples back with some comment that ways to use with SQLite vs Postgres as options.

Reply:
Oh yeah, same; I just noticed the Remarks section. 
To clarify, you think that we should show 2 different examples?